Foundation/NSURLSession/http/HTTPURLProtocol.swift

```diff
@@ -495,6 +498,16 @@ extension _HTTPURLProtocol: _EasyHandleDelegate {
         }
     }
 
+    fileprivate func notifyDelegate(aboutSentData count: Int64) {
```

Comment:
As the function is specific to upload tasks I wonder if this should be mentioned in the name too.